applications/app/controllers/ShortUrlsCampaignController.scala

```diff
@@ -1,33 +1,13 @@
 package controllers
 
-import common.{LinkTo, Logging}
+import common.{ExecutionContexts, LinkTo, Logging}
 import common.`package`._
+import campaigns.ShortCampaignCodes
 import conf.LiveContentApi
 import model.Cached
 import play.api.mvc.{RequestHeader, Action, Controller}
 
-import scala.concurrent.ExecutionContext.Implicits.global
-
-object ShortUrlsCampaignController extends Controller with Logging {
-
-  val campaigns = Map (
-    "iw" -> "twt_ipd",
-    "wc" -> "twt_wc",
-    "tf" -> "twt_fd",
-    "fb" -> "fb_gu",
-    "fo" -> "fb_ot",
-    "us" -> "fb_us",
-    "au" -> "soc_567",
-    "tw" -> "twt_gu",
-    "at" -> "twt_atn",
-    "sfb"-> "share_btn_fb",
-    "ip" -> "twt_iph",
-    "stw" -> "share_btn_tw",
-    "swa" -> "share_btn_wa",
-    "em" -> "email",
-    "sgp" -> "share_btn_gp",
-    "sbl" -> "share_btn_link"
-  )
+object ShortUrlsCampaignController extends Controller with Logging with ExecutionContexts {
```

Comment:
I meant to rename this as it fetches the full path for short urls with and without campaign codes. Shall we do it now?